### PR TITLE
Replace unnecessary double quotes in administrator #3

### PR DESCRIPTION
**Pull request by frankmayer**
### Summary of Changes
- Replace unnecessary double quotes

The changes in this PR should be fairly easy to review. In hope that this will get merged quickly so further work can be done without conflicting with other PRs. ;)

### Testing Instructions

None, should not change behavior

### Documentation Changes Required

None.

### CODE REVIEW

File: administrator/modules/mod_menu/menu.php [hunk 1]
```diff
@@ -119,7 +119,7 @@ public function renderMenu($id = 'menu', $class = '')
 		// Recurse through children if they exist
 		while ($this->_current->hasChildren())
 		{
-			echo "<ul " . $id . " " . $class . ">\n";
+			echo '<ul ' . $id . ' ' . $class . ">\n";
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_menu/menu.php]: hum why don't we use PHP_EOL const here ...

File: administrator/modules/mod_menu/menu.php [hunk 2]
```diff
@@ -214,20 +214,20 @@ public function renderLevel($depth)
 
 		if ($this->_current->link != null && $this->_current->target != null)
 		{
-			echo "<a" . $linkClass . " " . $dataToggle . " href=\"" . $this->_current->link . "\" target=\"" . $this->_current->target . "\" >"
-				. $this->_current->title . $dropdownCaret . "</a>";
+			echo '<a' . $linkClass . ' ' . $dataToggle . " href=\"" . $this->_current->link . "\" target=\"" . $this->_current->target . "\" >"
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_menu/menu.php]: why not use it in href and target too and remove the slashes?

File: administrator/modules/mod_menu/menu.php [hunk 3]
```diff
@@ -214,20 +214,20 @@ public function renderLevel($depth)
...
+				. $this->_current->title . $dropdownCaret . '</a>';
 		}
 		elseif ($this->_current->link != null && $this->_current->target == null)
 		{
-			echo "<a" . $linkClass . " " . $dataToggle . " href=\"" . $this->_current->link . "\">" . $this->_current->title . $dropdownCaret . "</a>";
+			echo '<a' . $linkClass . ' ' . $dataToggle . " href=\"" . $this->_current->link . "\">" . $this->_current->title . $dropdownCaret . '</a>';
```
Comment (andrepereiradasilva) [inline administrator/modules/mod_menu/menu.php]: same as above

File: administrator/templates/hathor/html/com_postinstall/messages/default.php [hunk 4]
```diff
@@ -12,15 +12,16 @@
...
+$param          = array(
+	'rssurl'          => 'https://www.joomla.org/announcements/release-news.feed?type=rss',
+	'rsstitle'        => 0,
+	'rssdesc'         => 0,
+	'rssimage'        => 1,
+	'rssitems'        => 5,
+	'rssitemdesc'     => 1,
+	'word_count'      => 200,
+	'cache'           => 0,
+	'moduleclass_sfx' => ' list-striped');
```
Comment (andrepereiradasilva) [inline administrator/templates/hathor/html/com_postinstall/messages/default.php]: cs: please move the `);` to the next line.

File: administrator/templates/isis/error.php [hunk 5]
```diff
@@ -122,7 +122,7 @@
 	<script src="<?php echo $this->baseurl; ?>/templates/<?php echo $this->template; ?>/js/template.js"></script>
 	<!--[if lt IE 9]><script src="<?php echo JUri::root(true); ?>/media/jui/js/html5.js"></script><![endif]-->
 </head>
-<body class="admin <?php echo $option . " view-" . $view . " layout-" . $layout . " task-" . $task . " ";?>" data-spy="scroll" data-target=".subhead" data-offset="87">
+<body class="admin <?php echo $option . ' view-' . $view . ' layout-' . $layout . ' task-' . $task . ' ';?>" data-spy="scroll" data-target=".subhead" data-offset="87">
```
Comment (andrepereiradasilva) [inline administrator/templates/isis/error.php]: the last space sounds really useful :)

**Review by andrepereiradasilva — APPROVED** [review]

File: administrator/modules/mod_menu/menu.php (hunk 1 above)
Comment (frankmayer) [inline administrator/modules/mod_menu/menu.php]: Depending on OS PHP_EOL sends \n (linux) or \r\n (windows) so unfortunately it's not doable...

File: administrator/modules/mod_menu/menu.php (hunk 2 above)
Comment (frankmayer) [inline administrator/modules/mod_menu/menu.php]: Wanted to leave that for a rainy day... Want PR's to go through as smoothly as possible ;)

File: administrator/modules/mod_menu/menu.php (hunk 3 above)
Comment (frankmayer) [inline administrator/modules/mod_menu/menu.php]: as above ;)

File: administrator/modules/mod_menu/menu.php (hunk 1 above)
Comment (andrepereiradasilva) [inline administrator/modules/mod_menu/menu.php]: ok

File: administrator/modules/mod_menu/menu.php (hunk 2 above)
Comment (andrepereiradasilva) [inline administrator/modules/mod_menu/menu.php]: well in this case i think this should be part of this PR. just saying ...

Comment (frankmayer) [inline administrator/modules/mod_menu/menu.php]: Well, ok then.. will do those as well... :)

**Review by andrepereiradasilva — APPROVED** [review]

File: administrator/includes/toolbar.php [hunk 6]
```diff
@@ -646,8 +646,8 @@ public static function versions($typeAlias, $itemId, $height = 800, $width = 500
 	public static function modal($targetModalId, $icon, $alt)
 	{
 		$title = JText::_($alt);
-		$dhtml = "<button data-toggle='modal' data-target='#" . $targetModalId . "' class='btn btn-small'>
-			<span class='" . $icon . "' title='" . $title . "'></span> " . $title . "</button>";
+		$dhtml = '<button data-toggle="modal" data-target="# ' . $targetModalId . '" class="btn btn-small">
```
Comment (shur) [inline administrator/includes/toolbar.php]: `data-target="# '` - this is not correct, you add an extra space here.

File: administrator/modules/mod_menu/menu.php [hunk 7]
```diff
@@ -214,20 +214,20 @@ public function renderLevel($depth)
 
 		if ($this->_current->link != null && $this->_current->target != null)
 		{
-			echo "<a" . $linkClass . " " . $dataToggle . " href=\"" . $this->_current->link . "\" target=\"" . $this->_current->target . "\" >"
-				. $this->_current->title . $dropdownCaret . "</a>";
+			echo '<a' . $linkClass . ' ' . $dataToggle . ' href="' . $this->_current->link . '" target="' . $this->_current->target . '" >'
```
Comment (shur) [inline administrator/modules/mod_menu/menu.php]: this extra space is not needed 
Please replase `'" >'` to `'">'` as it is done below.

**Review by shur — CHANGES_REQUESTED** [review]

File: administrator/includes/toolbar.php (hunk 6 above)
Comment (frankmayer) [inline administrator/includes/toolbar.php]: Whoops! Yes ;)

File: administrator/modules/mod_menu/menu.php (hunk 7 above)
Comment (frankmayer) [inline administrator/modules/mod_menu/menu.php]: OK, done.

**Comment by shur** [comment]
I have tested this item :white_check_mark: successfully on a9cf8ed5f15342f09199dd2a8ff07a522fb48564<br /><br />Now is good!<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/13295">issues.joomla.org/tracker/joomla-cms/13295</a>.</sub>

**Review by shur — APPROVED** [review]

**Review by andrepereiradasilva — APPROVED** [review]

**Comment by shur** [comment]
@frankmayer 
**You must see this:** 
https://www.facebook.com/groups/joomlanospam/permalink/10153990199490997/

**Comment by frankmayer** [comment]
@shur Thank you for the link :) It's very nice to get a personal acknowledgement. But, lets not forget the community effort involved, to get any PR finally merged.

So, thank you team Joomla, for constructive criticism, testing and improving my commits (and me).

**Pull request merged**